src/cmd/cli/command/stack.go

```diff
@@ -0,0 +1,175 @@
+package command
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/AlecAivazis/survey/v2"
+	cliClient "github.com/DefangLabs/defang/src/pkg/cli/client"
+	"github.com/DefangLabs/defang/src/pkg/modes"
+	"github.com/DefangLabs/defang/src/pkg/stacks"
+	"github.com/DefangLabs/defang/src/pkg/term"
+	"github.com/spf13/cobra"
+)
+
+func makeStackCmd() *cobra.Command {
+	var stackCmd = &cobra.Command{
+		Use:   "stack",
+		Args:  cobra.NoArgs,
+		Short: "Manage Defang deployment stacks",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+	}
+	stackNewCmd := makeStackNewCmd()
+	stackCmd.AddCommand(stackNewCmd)
+	stackListCmd := makeStackListCmd()
+	stackCmd.AddCommand(stackListCmd)
+	stackRemoveCmd := makeStackRemoveCmd()
+	stackCmd.AddCommand(stackRemoveCmd)
+	return stackCmd
+}
+
+func makeStackNewCmd() *cobra.Command {
+	var stackNewCmd = &cobra.Command{
+		Use:   "new STACK_NAME",
+		Args:  cobra.ExactArgs(1),
```

Comment:
We ask all other params interactively, so why not the stack name? (like `pulumi stack init`